admin/class-customizer.php

```diff
@@ -0,0 +1,324 @@
+				$this->wp_customize, 'wpseo-breadcrumbs-home', array(
+					'label'       => __( 'Anchor Text for Homepage:', 'wordpress-seo' ),
+					'type'        => 'text',
+					'section'     => 'wpseo_breadcrumbs_customizer_section',
+					'settings'    => 'wpseo_internallinks[breadcrumbs-home]',
+					'context'     => '',
+					'active_callback' => array( $this, 'breadcrumbs_active_callback' ),
+				)
+			)
+		);
+	}
+
+	/**
+	 * Adds the breadcrumbs prefix text field
+	 */
+	private function breadcrumbs_prefix_setting() {
+
+		$this->wp_customize->add_setting(
+			'wpseo_internallinks[breadcrumbs-prefix]', array(
+				'default'   => '',
+				'type'      => 'option',
+				'transport' => 'refresh',
+			)
+		);
+
+		$this->wp_customize->add_control(
+			new WP_Customize_Control(
+				$this->wp_customize, 'wpseo-breadcrumbs-prefix', array(
+					'label'       => __( 'Prefix for the breadcrumb path:', 'wordpress-seo' ),
+					'type'        => 'text',
+					'section'     => 'wpseo_breadcrumbs_customizer_section',
+					'settings'    => 'wpseo_internallinks[breadcrumbs-prefix]',
+					'context'     => '',
+					'active_callback' => array( $this, 'breadcrumbs_active_callback' ),
+				)
+			)
+		);
+	}
+
+	/**
+	 * Adds the breadcrumbs archive prefix text field
+	 */
+	private function breadcrumbs_archiveprefix_setting() {
+
+		$this->wp_customize->add_setting(
+			'wpseo_internallinks[breadcrumbs-archiveprefix]', array(
+				'default'   => '',
+				'type'      => 'option',
+				'transport' => 'refresh',
+			)
+		);
+
+		$this->wp_customize->add_control(
+			new WP_Customize_Control(
+				$this->wp_customize, 'wpseo-breadcrumbs-archiveprefix', array(
+					'label'       => __( 'Prefix for Archive breadcrumbs:', 'wordpress-seo' ),
+					'type'        => 'text',
+					'section'     => 'wpseo_breadcrumbs_customizer_section',
+					'settings'    => 'wpseo_internallinks[breadcrumbs-archiveprefix]',
+					'context'     => '',
+					'active_callback' => array( $this, 'breadcrumbs_active_callback' ),
+				)
+			)
+		);
+	}
+
+	/**
+	 * Adds the breadcrumbs search prefix text field
+	 */
+	private function breadcrumbs_searchprefix_setting() {
+
+		$this->wp_customize->add_setting(
+			'wpseo_internallinks[breadcrumbs-searchprefix]', array(
+				'default'   => '',
+				'type'      => 'option',
+				'transport' => 'refresh',
+			)
+		);
+
+		$this->wp_customize->add_control(
+			new WP_Customize_Control(
+				$this->wp_customize, 'wpseo-breadcrumbs-searchprefix', array(
+					'label'       => __( 'Prefix for Search Page breadcrumbs:', 'wordpress-seo' ),
+					'type'        => 'text',
+					'section'     => 'wpseo_breadcrumbs_customizer_section',
+					'settings'    => 'wpseo_internallinks[breadcrumbs-searchprefix]',
+					'context'     => '',
+					'active_callback' => array( $this, 'breadcrumbs_active_callback' ),
+				)
+			)
+		);
+
+	}
+
+	/**
+	 * Adds the breadcrumb 404 prefix text field
+	 */
+	private function breadcrumbs_404_setting() {
+
```

Comment:
Why this extra new line here? And in multiple other spots after the function declaration? Surprised the code style doesn't whine about it.